andela/trime
# Feature/dashboard e2e

#### What does this PR do?
Adds end to end tests for creating organisations and projects.

#### Description of Task to be completed?
Write e2e tests using Protractor and Selenium to test the dashboard workflow.

#### How should this be manually tested?
Clone this repo, install the dependencies by running
```
npm install
```
Make sure you have Java (JDK) installed in your system then run:
```
npm run e2esetup
```
This will install Selenium weddriver if it doesn't exist in your system.

Then run:
```
npm run test:e2e
```

#### What are the relevant pivotal tracker stories?
[End to End Tests for Dashboard](https://www.pivotaltracker.com/projects/1454736/stories/118428823)

@hisabimbola @andela-ekupara @andela-kndungu @gottsohn
